Godeps/_workspace/src/k8s.io/kubernetes/pkg/kubectl/cmd/get.go

```diff
@@ -323,3 +355,32 @@ func RunGet(f *cmdutil.Factory, out io.Writer, cmd *cobra.Command, args []string
 	}
 	return utilerrors.NewAggregate(allErrs)
 }
+
+// mustPrintWithKinds determines if printer output
+// should print multiple kinds of objects, even if
+// "all-kinds" flag is not set
+func mustPrintWithKinds(objs []runtime.Object, infos []*resource.Info, sorter *kubectl.RuntimeSort) bool {
+	var lastMap *meta.RESTMapping
+	var withKind bool
+
+	if withKind {
```

Comment:
This if statement is dead code, `withKind` was just declared a line above, and it is always false at this point.